### PR TITLE
Added Example playbook and var description

### PR DESCRIPTION
**Pull request by Scriptkiddi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,34 @@
-# Role Name
+# Git repsoitory deploy
```
Comment (SlothOfAnarchy) [inline README.md]: *repository

Also, the repo name is `static_web` :P

File: README.md [hunk 2]
```diff
@@ -1,25 +1,34 @@
...
 
-you need the nginx role
-
+None 
 
 ## Role Variables
 
+| Variable name | Required | Description |
+| -------- | -------- | -------- |
+| `static_web_folder_to_copy`     | :heavy_check_mark:     | The variable contains every pair of `src` and `to` that is needed to deploy a git repository     |
```
Comment (SlothOfAnarchy) [inline README.md:14]: From the example I gather that `src` may be a git repo address. Is https also allowed?

File: README.md [hunk 3]
```diff
@@ -1,25 +1,34 @@
...
+    to: /var/www/uno
+  - src: git@github.com:stuvusIT/www_campusbeach
+    to: /var/www/campusbeach
+  - src: git@github.com:stuvusIT/www_fsen_paedagogik
+    to: /var/www/paedagogik
+  - src: git@github.com:stuvusIT/www_fsen_sozialwissenschaften
+    to: /var/www/www_fsen_sozialwissenschaften
+  - src: git@github.com:stuvusIT/www_ak_zeitung
+    to: /var/www/zeitung
 ```
```
Comment (SlothOfAnarchy) [inline README.md:35]: Your author link isn't working.

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

**Comment by SlothOfAnarchy** [comment]
Also, please add tags and a description on the github repo page.

**Comment by Scriptkiddi** [comment]
all done pls rereview @SlothOfAnarchy

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**